app/components/Controls.tsx

```diff
@@ -46,21 +53,23 @@ export const Controls = ({
     (e: any) => {
       handleSubmit(e);
       stopAudio();
+      e.target.value = '';
+      handleInputChange(e)
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps 
     [stopAudio, handleSubmit]
   );
 
   return (
-    <form onSubmit={submitter}>
+    <form onSubmit={submitter} ref={formRef}>
       <div className="relative">
         <div className="absolute w-full -top-[4.5rem] py-4 flex justify-between">
           <Settings />
           <Download messages={messages} />
         </div>
         <div className="flex bg-[#101014] rounded-full">
           <span
-            className={`rounded-s-full ps-0.5 py-0.5 ${
+            className={`rounded-tl-md rounded-bl-md ps-0.5 py-0.5 ${
```

Comment:
If we know the height of this input, is it possible to keep the full rounded radius, and just have the sides increase. i.e. 4rem high, 2rem radius?

```suggestion
            className={`rounded-tl-[2rem] rounded-bl-[2rem] ps-0.5 py-0.5 ${
```